app/brave_main_delegate.cc

```diff
@@ -139,17 +140,15 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
       extensions_features::kNewExtensionUpdaterService.name,
 #endif
       features::kDesktopPWAWindowing.name,
+      omnibox::kSimplifyHttpsIndicator.name,
   };
 
   // Disabled features.
   const std::unordered_set<const char*> disabled_features = {
       autofill::features::kAutofillSaveCardSignInAfterLocalSave.name,
       autofill::features::kAutofillServerCommunication.name,
-      features::kAudioServiceOutOfProcess.name,
-      features::kDefaultEnableOopRasterization.name,
```

Comment:
Same as: https://github.com/brave/brave-core/pull/2471/files#r286683268 

Feature is disabled here: https://cs.chromium.org/chromium/src/gpu/config/gpu_finch_features.cc?l=58

cc: @mkarolin, @bbondy